service/scaladsl/server/src/main/scala/com/lightbend/lagom/scaladsl/server/LagomServer.scala

```diff
@@ -46,6 +46,17 @@ object LagomServer {
       }
     }
   }
+
+  /**
+   * Internal API used by AST generated by the serverFor macro.
+   */
+  def forService(binding: LagomServiceBinding[_]): LagomServer = {
```

Comment:
No, because then the generated macro AST won't compile because it will live in the users package.